src/crypto/tls/common.go

```diff
@@ -1296,6 +1350,10 @@ type Certificate struct {
 	// SignedCertificateTimestamps contains an optional list of Signed
 	// Certificate Timestamps which will be served to clients that request it.
 	SignedCertificateTimestamps [][]byte
+	// DelegatedCredential is a serialized Delegated Credential, signed by
```

Comment:
Mention that it is optional and nil if not included.

Reply:
Added as well.